vmdb/app/views/ops/_schedule_form_timer.html.haml

```diff
@@ -0,0 +1,61 @@
+- url = url_for(:action =>'schedule_form_field_changed', :id=>"#{@schedule.id || "new"}")
+#form_timer_div
+  %p.legend Timer
+  %table.style1
+    %tr
+      %td.key Run
+      %td
+        = select_tag('timer_typ',
+          options_for_select(["Once","Hourly","Daily","Weekly","Monthly"],
+          @edit[:new][:timer_typ]),
+          "data-miq_sparkle_on"=>true,
+          "data-miq_observe"=>{:url=>url}.to_json)
+        = hidden_span_if(@edit[:new][:timer_typ].downcase != "daily", :id=>"daily_span" )
```

Comment:
can we cache the
`>{:url=>url}.to_json` and the `@edit[:new][:timer_typ].downcase`, please?